### ISSUE TITLE
"Download for later" doesn't seem to work

### ISSUE THREAD

**Issue by chau-intl** [mapswipe/mapswipe#64]
I recently wanted to download a chunk of tiles to help me pass the time on a flight. When I select "Download for later" under a project it first asks me for permission to download without WiFi. After giving the permission the "download for later" button is greyed out and nothing seem to happen. It doesn't seem to matter which package size I select. I kept the phone and mapswipe open for more than 10 minutes and still nothing happened. 

I have never been able to download for later.
I have just reinstalled mapswipe and it didn't help.

**Comment by laurentS** [comment]
Hello @chau-intl, Thanks for the bug report! We're very much aware of this issue, it's been reported by dozens of people outside this bug tracker. Weirdly enough there doesn't seem to be a ticket for it. So I'll make this ticket the official one for "offline support" :)

I have a fix in the pipeline, that is part of https://github.com/orgs/mapswipe/projects/2
I will track progress on this ticket, so stay tuned!

**Comment by chau-intl** [comment]
Hi @laurentS, thanks for replying. I wondered why I couldn't find a ticket on the issue :)

I look forward to test the fix. Are there a beta group on Google play or is it released as apk?

**Comment by laurentS** [comment]
We're working on getting the update into the playstore, but for now, you can test the current `dev` branch build from https://github.com/mapswipe/mapswipe/releases You're very welcome to try it out, and report back here :) (note that this build works with a dev server, so your contributions will be discarded randomly, and you'll need to create a different account if you want to use it).

**Comment by tkas** [comment]
Hello, what is the status of this? Project for 2.0 is closed, this seems to end in todo list unfinished.

**Comment by laurentS** [comment]
@tkas you're right that it's not done yet. I think this now also relates to https://github.com/mapswipe/mapswipe/issues/157 as it could amend the way we fix this particular issue. Any thoughts welcome!

**Comment by tkas** [comment]
OK, if giving all data to Google is the way this app go, uninstalling it is the way I prefer.
